melodies_monet/driver.py

```diff
@@ -1731,7 +1786,8 @@ def plotting(self):
                                                                         "sat_grid_sfc", "sat_grid_clm", 
                                                                         "sat_swath_prof"]: 
                             # xarray doesn't need nan drop because its math operations seem to ignore nans
-                            pairdf = pairdf_all
+                            # MEB (10/9/24): Add statement to ensure model and obs variables have nans at the same place
+                            pairdf = pairdf_all.where(pairdf_all[obsvar].notnull())
```

Comment:
Technically the model could also have nulls in places where the obs doesn't.
```suggestion
                            # MEB (10/9/24): Add statement to ensure model and obs variables have nans at the same place
                            pairdf = pairdf_all.where(pairdf_all[obsvar].notnull() & pairdf_all[modvar].notnull())
```